sources/packages/backend/apps/db-migrations/src/sql/Reports/Update-program-and-offering-status-report.sql

```diff
@@ -0,0 +1,156 @@
+UPDATE
+  sims.report_configs
+SET
+  report_sql = (
+    'WITH offerings_dataset AS (
+      SELECT
+        institution_locations.name AS "Institution Location Name",
+        education_programs.id AS "program_id",
+        education_programs.program_name AS "Program Name",
+        education_programs.sabc_code AS "SABC Program Code",
+        education_programs.regulatory_body AS "Regulatory Body",
+        education_programs.credential_type AS "Credential Type",
+        education_programs.completion_years AS "Program Length",
+        CASE
+          WHEN (
+            education_programs.delivered_on_site = true
+            AND education_programs.delivered_online = true
+          ) THEN ''Blended''
+          WHEN education_programs.delivered_on_site = true THEN ''On-site''
+          WHEN education_programs.delivered_online = true THEN ''Online''
+        END AS "Delivery",
+        education_programs.course_load_calculation AS "Credit or Hours",
+        education_programs.program_status AS "Program Status",
+        CASE
+          WHEN education_programs.is_active = true THEN false
+          WHEN education_programs.is_active = false THEN true
+        END AS "Program Deactivated?",
+        CAST(
+          education_programs.effective_end_date AS varchar
+        ) AS "Program Expiry Date",
+        education_programs_offerings.offering_name AS "Offering Name",
+        CAST(
+          education_programs_offerings.study_start_date AS varchar
+        ) AS "Study Start Date",
+        CAST(
+          education_programs_offerings.study_end_date AS varchar
+        ) AS "Study End Date",
+        education_programs_offerings.year_of_study AS "Year of Study",
+        education_programs_offerings.offering_type AS "Offering Type",
+        education_programs_offerings.offering_status AS "Offering Status",
+        education_programs_offerings.offering_intensity AS "Offering Intensity"
+      FROM
+        sims.education_programs education_programs
+        LEFT JOIN sims.education_programs_offerings education_programs_offerings ON education_programs.id = education_programs_offerings.program_id
+        INNER JOIN sims.institution_locations institution_locations ON education_programs_offerings.location_id = institution_locations.id
+        INNER JOIN sims.institutions institutions ON institution_locations.institution_id = institutions.id
+      WHERE
+        institutions.id = :institution
+        AND education_programs_offerings.study_start_date >= :startDate
+        AND education_programs_offerings.study_start_date <= :endDate
```

Comment:
I missing the motivation for using this instead of the `between` as it was before, can you please clarify?